feedback-module/src/components/TextboxList.js

```diff
@@ -5,52 +5,47 @@ import { useTranslation } from "react-i18next";
 import Textbox from "./common/Textbox";
 import ErrorAlert from "./common/ErrorAlert";
 
-function TextboxList({ inputs, inputQuestions, setInputQuestions, className }) {
+function TextboxList({
+  inputs,
+  inputQuestions,
+  setInputQuestions,
+  className,
+  inputRefs,
+}) {
   const { t } = useTranslation();
 
-  const onChange = ({ target }) => {
-    let answers = inputQuestions;
-    answers[target.id].answer = target.value;
-    setInputQuestions(answers);
```

Comment:
What was the purpose of this change?

Reply:
We used to have the id for each input just be its index, which could be problematic if the host website also has the same id's somewhere on their website (id's are meant to be unique), so I decided to make the id more specific to the feedback-module (feedback-input-${index}) and less likely to be a duplicate id on a host website. As a result, we can't just pull the index directly from target.id anymore so I needed to make it a curried function instead.

Reply:
Ok got it but I still don't think it needs to be a curried function?

Reply:
Hmmm since we need the index I think it either has to be curried or we have to move the function to the body of inputs .map((input, index) -> {...}). Do you think the latter option would be cleaner?

Reply:
Wait why would we need to move it inside the inputs.map